pkg/cs/util.go

```diff
@@ -89,3 +90,11 @@ func ArgToEnvVarMap(input []string) (map[string]string, error) {
 	}
 	return res, nil
 }
+
+func ValidateUrl(urlIn string) (string, error) {
+	urlParsed, err := url.Parse(urlIn)
```

Comment:
we should also validate that this is a `http(s)` url, as we only support those.
Just for better validation

Reply:
done